src/components/homePage/About.vue

```diff
@@ -31,68 +20,33 @@
 	margin: auto;
 	color: black;
 }
-@media screen and (max-width:1250px) {
+.about_section{
+	display: flex;
+	flex-direction: row;
+}
+.about_image{
+	width: 80vh;
```

Comment:
Does this width parameter do anything? I think flex is controlling it's width. It looks unchanged when I manually remove it. Also, in general it's better to use percentage rather than vh :)
![image](https://user-images.githubusercontent.com/41529410/119239248-cf350d80-bb15-11eb-9cfd-4c126ed81cc6.png)

Reply:
Yes I guess width is playing it's role...I have put two different pictures with and without width parameter. 

With width

![withWidth](https://user-images.githubusercontent.com/78088990/120039060-406f3780-bfd2-11eb-8b6a-f5a487e3d923.JPG)

Without width

![withoutWidth](https://user-images.githubusercontent.com/78088990/120039253-95ab4900-bfd2-11eb-8688-d3811340f265.JPG)


I have changes the unit to percentage in this pictures. But this is the difference. I can remove it if you want. Let me know.